build.gradle

```diff
@@ -59,35 +50,30 @@ dependencies {
     testCompile 'org.spockframework:spock-core:1.0-groovy-2.4'
     testCompile 'org.springframework.boot:spring-boot-starter-test:' + springBootVersion
 }
+compileJava.dependsOn(processResources)
 
-task wrapper(type: Wrapper) {
-    gradleVersion = '2.3'
-}
+task wrapper(type: Wrapper) { gradleVersion = '2.3' }
 
-checkstyle {
-    toolVersion = '6.4.1'
-}
+checkstyle { toolVersion = '6.4.1' }
 
 codenarc {
     toolVersion = '0.23'
     configFile = new File("${projectDir}/config/codenarc/test-rules.groovy")
 }
 
-nexusStaging {
-    packageGroup = "pl.allegro"
-}
+nexusStaging { packageGroup = 'pl.allegro' }
 
 modifyPom {
     project {
         name 'Spring Boot Starter Handlebars'
         description 'Spring Boot auto-configuration for Handlebars template engine'
-        url 'https://github.com/allegro/spring-boot-starter-handlebars'
+        url 'https://github.com/allegro/handlebars-spring-boot-starter'
         inceptionYear '2015'
 
         scm {
-            url 'https://github.com/allegro/spring-boot-starter-handlebars'
-            connection 'scm:git@github.com:allegro/spring-boot-starter-handlebars.git'
-            developerConnection 'scm:git@github.com:allegro/spring-boot-starter-handlebars.git'
+            url 'https://github.com/allegro/handlebars-spring-boot-starter'
```

Comment:
To rename or not to rename, that is the question ;) I guess we don't have to rename repo and it would be better for seo but on the other hand it's a good practice to have repo name consistent with artifact name.